src/server/pps/cmds/cmds_test.go

```diff
@@ -1527,7 +1527,7 @@ func TestInspectClusterDefaults(t *testing.T) {
 		}, nil
 	})
 	require.NoError(t, tu.PachctlBashCmd(t, env.PachClient, `
-		pachctl inspect defaults --cluster | match '{}'
+		pachctl inspect defaults --cluster | match '{.*}'
```

Comment:
Should we check for the default values as well?

Reply:
I chose not to, because the order is not constant.